src/v/cloud_storage/topic_manifest.cc

```diff
@@ -154,6 +157,7 @@ struct topic_manifest_handler
     std::optional<int32_t> _partition_count;
     std::optional<int16_t> _replication_factor;
     std::optional<model::initial_revision_id> _revision_id{};
+    std::optional<model::initial_revision_id> _prev_revision_id{};
```

Comment:
nit: just wondering what purpose using a std::optional has here. from what i can tell any time the revision doesn't have a value its given a value of ::min() so it could be initialized directly to ::min?

Reply:
This would definitely work. But I just decided to follow current approach. Otherwise my field would be out of place, and the reader of the code will definitely have questions like "why everything is optional except this field".